src/raftstore/store/util.rs

```diff
@@ -256,9 +256,10 @@ pub fn check_region_epoch(
             | AdminCmdType::InvalidAdmin
             | AdminCmdType::ComputeHash
             | AdminCmdType::VerifyHash => {}
-            AdminCmdType::Split | AdminCmdType::BatchSplit => check_ver = true,
```

Comment:
why does Split need to be checked ConfVer?

Reply:
The test case did fail with error message `expected peers are 3, but got 2 from request`. So there must be some cases that PD gives TiKVs a `Split` command with lower conf version. However I can't find it, so I just add a unit test for this, instead of an integration test.